src/components/structures/ToastContainer.js

```diff
@@ -74,7 +74,7 @@ export default class ToastContainer extends React.Component {
         });
 
         return (
-            <div className={containerClasses}>
+            <div className={containerClasses} role="alert" aria-live="polite">
```

Comment:
In addition to what @t3chguy said: role="alert" implies aria-live="assertive". You want this to be spoken in an assertive manner, so putting aria-live="polite" contradicts the purpose of role="alert". I'd just omit the aria-live attribute here, but keep the other one that turns it off for the individual buttons.